sys/net/gnrc/network_layer/sixlowpan/frag/rbuf.c

```diff
@@ -29,8 +29,15 @@
 #define ENABLE_DEBUG    (0)
 #include "debug.h"
 
+/* estimated fragment payload size to determinate RBUF_INT_SIZE, default to
+ * frame size - fragment header. */
+#ifndef GNRC_SIXLOWPAN_FRAG_SIZE
+#define GNRC_SIXLOWPAN_FRAG_SIZE (102 - 5)
```

Comment:
127 includes MAC header:

```
127
  - 2 (Frame Control)
  - 1 (Sequence Number)
  - 2 (Destination PAN Identifier)
  - 8 (Destination Address)
  - 2 (Source PAN Identifier)
  - 8 (Source Address)
  - 2 (FCS)
= 102
```

If security header is enabled, it may takes additional 14 bytes, so `GNRC_SIXLOWPAN_FRAG_SIZE` must be changed to `(88 - 5)`.